running benchmarks fails - Could not find or load main class vavr.JmhRunner

I try to execute this as desribed in the readme.md

```bash
~/vavr $ mvn --version
Apache Maven 3.5.0 (ff8f5e7444045639af65f6095c62210b5713f426; 2017-04-03T21:39:06+02:00)
Maven home: C:\apache-maven-3.5.0
Java version: 1.8.0_121, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0\jre
Default locale: en_GB, platform encoding: Cp1252
OS name: "windows 7", version: "6.1", arch: "amd64", family: "windows"
~/vavr $ mvn clean test -P benchmark -pl vavr-benchmark
.
.
.
[DEBUG] Executing command line: [C:\ProgramData\Oracle\Java\javapath\java.exe, -classpath, C:\cygwin64\home\r892107\sourcecode\vavr\vavr-benchmark\target\test-classes;C:\cygwin64\home\r892107\sourcecode\vavr\vavr-benchmark\target\classes;C:\Users\r892107\.m2\repository\i
o\vavr\vavr\1.0.0-SNAPSHOT\vavr-1.0.0-SNAPSHOT.jar;C:\Users\r892107\.m2\repository\io\vavr\vavr-match\1.0.0-SNAPSHOT\vavr-match-1.0.0-SNAPSHOT.jar;C:\Users\r892107\.m2\repository\org\scalaz\scalaz-core_2.12\7.3.0-M8\scalaz-core_2.12-7.3.0-M8.jar;C:\Users\r892107\.m2\repo
sitory\org\scala-lang\scala-library\2.12.1\scala-library-2.12.1.jar;C:\Users\r892107\.m2\repository\org\clojure\clojure\1.9.0-alpha14\clojure-1.9.0-alpha14.jar;C:\Users\r892107\.m2\repository\org\pcollections\pcollections\2.1.2\pcollections-2.1.2.jar;C:\Users\r892107\.m2
\repository\org\functionaljava\functionaljava-java8\4.6\functionaljava-java8-4.6.jar;C:\Users\r892107\.m2\repository\org\functionaljava\functionaljava\4.6\functionaljava-4.6.jar;C:\Users\r892107\.m2\repository\org\eclipse\collections\eclipse-collections\8.0.0\eclipse-col
lections-8.0.0.jar;C:\Users\r892107\.m2\repository\org\eclipse\collections\eclipse-collections-api\8.0.0\eclipse-collections-api-8.0.0.jar;C:\Users\r892107\.m2\repository\net\jcip\jcip-annotations\1.0\jcip-annotations-1.0.jar;C:\Users\r892107\.m2\repository\org\openjdk\j
ol\jol-core\0.7.1\jol-core-0.7.1.jar;C:\Users\r892107\.m2\repository\org\jetbrains\annotations\15.0\annotations-15.0.jar;C:\Users\r892107\.m2\repository\junit\junit\4.12\junit-4.12.jar;C:\Users\r892107\.m2\repository\org\hamcrest\hamcrest-core\1.3\hamcrest-core-1.3.jar;C
:\Users\r892107\.m2\repository\org\openjdk\jmh\jmh-generator-annprocess\1.19\jmh-generator-annprocess-1.19.jar;C:\Users\r892107\.m2\repository\org\openjdk\jmh\jmh-core\1.19\jmh-core-1.19.jar;C:\Users\r892107\.m2\repository\net\sf\jopt-simple\jopt-simple\4.6\jopt-simple-4
.6.jar;C:\Users\r892107\.m2\repository\org\apache\commons\commons-math3\3.2\commons-math3-3.2.jar, vavr.JmhRunner]
Error: Could not find or load main class vavr.JmhRunner
[ERROR] Command execution failed.
.
.
.
```

This fails `Could not find or load main class vavr.JmhRunner`

--- a/pom.xml
+++ b/pom.xml
@@ -483,7 +483,7 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
                                     <arguments>
                                         <argument>-classpath</argument>
                                         <classpath />
-                                        <argument>vavr.JmhRunner</argument>
+                                        <argument>io.vavr.JmhRunner</argument>
                                     </arguments>
                                 </configuration>
                             </execution>

